service-front/module/Application/view/application/pages/vouching/how_will_you_confirm.twig

```diff
@@ -0,0 +1,286 @@
+{% extends "layout/layout" %}
+
+{% block beforeMain %}
+    {% include 'layout/id_check_banner.twig' with details_data %}
```

Comment:
no name is showing in the banner because in the vouching route the name of the donor is stored in the `vouching_for` sub-array of details_data (and the name at this point is not populated).

I've handled this in the [confirm_vouching page](https://github.com/ministryofjustice/opg-paper-identity/blob/main/service-front/module/Application/view/application/pages/vouching/confirm_vouching.twig#L4-L5)

Reply:
Thats really helpful. Thanks @davidbridgwood